vss-cats/src/main/scala/com/virtuslab/vss/cats/stats/grpc/services/StatsGrpcService.scala

```diff
@@ -0,0 +1,25 @@
+package com.virtuslab.vss.cats.stats.grpc.services
+
+import cats.effect.*
+import cats.*
+import cats.syntax.all.*
+import io.grpc.*
+import fs2.grpc.syntax.all.*
+import com.virtuslab.vss.proto.cats.password.*
+import com.virtuslab.vss.common.*
+import com.virtuslab.vss.cats.stats.services.*
+
+object StatsGrpcService {
+  def make[F[_]: Async: Monad](stats: Stats[F]): Resource[F, ServerServiceDefinition] =
+    StatsServiceFs2Grpc.bindServiceResource[F](new StatsServiceFs2Grpc {
+      override def getLatestEvents(request: EmptyRequest, ctx: Metadata): F[LatestEvents] =
+        stats.getLatestEvents(100)
+          .map(_.map {
+            case event @ Event.CheckedPwned(passwordHash) =>
+              EventResponse(eventType = event.getClass().getCanonicalName(), passwordHash = Some(passwordHash))
```

Comment:
[minor] It's possible to use Cats syntax sugar here. I think it makes sense in demo project like this. It's not a big deal, but I believe it's easier to read.

```
import cats.syntax.option._
... passwordHash = passwordHash.some
```

In other places in this file and in other files too - just search for `Some(` in this module.

Reply:
I don't see any benefit from this